tests/testthat/test-fmt_symbol_first.R

```diff
@@ -1,33 +1,25 @@
-# Function to skip tests if Suggested packages not available on system
-check_suggests <- function() {
-  skip_if_not_installed("rvest")
-  skip_if_not_installed("xml2")
+test_gt_by_col <- function(col_n, row_first = TRUE, expectation) {
+  check_suggests()
```

Comment:
Top-level code, outside of `test_that()`, is something we're trying to minimize.

But given that you use `ex_html_tab` in several tests, the best compromise seems to be to bundle the skip and the data creation inside this file-scoped helper.